content/exception-filters.md

```diff
@@ -292,30 +294,35 @@ import {
   HttpException,
   HttpStatus,
 } from '@nestjs/common';
+import { AbstractHttpAdapter, HttpAdapterHost } from '@nestjs/core';
 
 @Catch()
 export class AllExceptionsFilter implements ExceptionFilter {
-  catch(exception: unknown, host: ArgumentsHost) {
+  private readonly httpAdapter: AbstractHttpAdapter;
+
+  constructor(httpAdapterHost: HttpAdapterHost) {
+    this.httpAdapter = httpAdapterHost.httpAdapter;
```

Comment:
In certain situations, `httpAdapter` might not be yet available in a constructor so I'd suggest retrieving it within the `catch` method instead.

Reply:
good to know.

![image](https://user-images.githubusercontent.com/13461315/141979789-4d643f86-4e04-475b-8ade-688d53ec63c0.png)